### PR TITLE
1

### PR DESCRIPTION
**Pull request by huji405**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

### Description
Added new tutorial to LUA section
### Why is this valuable (or not)?
It teaches English speakers LUA
### How do we know it's really free?
It's a Youtube video
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -376,6 +376,7 @@
 
 * [Cloud Computing and Development](https://uaceit.com/courses/cloud-computing-and-development) - Ashwin Kumar Ramaswamy (UAceIt) (email address *required*)
 * [Cloud Computing Full Course In 11 Hours \| Cloud Computing Tutorial For Beginners](https://www.youtube.com/watch?v=2LaAJq1lB1Q) - Edureka!
+* [Cloud Computing Tutorial for Beginners - Cloud Computing Explained - Cloud Computing](https://www.youtube.com/watch?v=RWgW-CgdIk0) - Simplilearn
```
Comment (eshellman) [inline courses/free-courses-en.md:379]: For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -1155,6 +1156,7 @@
 
 ### Lua
 
+* [Full Lua Programming Crash Course - Beginner to Advanced](https://www.youtube.com/watch?v=1srFmjt1Ib0) - Steve's teacher
```
Comment (eshellman) [inline courses/free-courses-en.md:1159]: For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

**Review by eshellman — CHANGES_REQUESTED** [review]

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -1155,6 +1155,7 @@
 
 ### Lua
 
+* [Lua Programming Tutorials](https://youtube.com/playlist?list=PLYBJzqz8zpWavt37pA6NANJTGStIHpybU&si=DtatTyXBfJhH82CA) - Steve's teacher
```
Comment (eshellman) [inline courses/free-courses-en.md:1158]:
```suggestion
* [Lua Programming Tutorials](https://youtube.com/playlist?list=PLYBJzqz8zpWavt37pA6NANJTGStIHpybU) - Steve's teacher
```

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Comment by huji405** [comment]
Please check my other branch named "3 changed" and leave a review. tysm

**Comment by Thenlie** [comment]
@huji405 I looked at your other PR's and they seem unrelated to this. So I think this is still valid? Feel free to close it out if not.

**Pull request merged**